content/developer/composer/blueprint-creation.md

```diff
@@ -62,8 +62,25 @@ Click the node to display the node property panel. It includes the following com
 
 To close the panel, press **Esc** or click the close window icon on the top-right corner.
 
-![Working with Nodes]( /images/composer/working-with-nodes.png )
+![](/images/composer/properties-pane.png "Properties pane")
```

Comment:
Didn't now that it's possible to specify tooltip this way. Probably we should use it more often.

Alt name not specified intentionally?

Reply:
Yes, I could use the same string for title and description, but I don't think it would make sense.